### PR TITLE
3796 add judges to case deadlines

### PR DESCRIPTION
**Pull request by rachelschneiderman**

### CODE REVIEW

File: web-client/src/presenter/computeds/caseDeadlineReportHelper.js [hunk 1]
```diff
@@ -51,10 +55,13 @@ export const caseDeadlineReportHelper = (get, applicationContext) => {
     }
   };
 
+  const judgeFilter = get(state.screenMetadata.caseDeadlinesFilter.judge);
+
   caseDeadlines = caseDeadlines
     .sort(sortByDateAndDocketNumber(applicationContext))
     .map(d => ({
       ...d,
+      associatedJudgeFormatted: d.associatedJudge.replace(/^Judge\s+/, ''),
```
Comment (matthopson) [inline web-client/src/presenter/computeds/caseDeadlineReportHelper.js]: What do you think about a little utility function that does this `Judge` replacement?

**Review by matthopson — APPROVED** [review]
One suggestion that you could work on in another PR (or not at all).

File: web-client/src/presenter/computeds/caseDeadlineReportHelper.js (hunk 1 above)
Comment (rachelschneiderman) [reply web-client/src/presenter/computeds/caseDeadlineReportHelper.js]: I like it!

File: web-client/src/views/CaseDeadlines/CaseDeadlines.jsx [hunk 2]
```diff
@@ -33,6 +36,31 @@ export const CaseDeadlines = connect(
...
+                </div>
+                <div className="grid-col-3">
+                  <BindedSelect
+                    ariaDescribedBy="case-deadlines-tab filterHeading"
+                    ariaLabel="judge"
+                    bind="screenMetadata.caseDeadlinesFilter.judge"
+                    className="select-left"
+                    id="judgeFilter"
+                    name="judge"
+                    placeHolder="xyz"
```
Comment (kkoskelin) [inline web-client/src/views/CaseDeadlines/CaseDeadlines.jsx]: Is this actually the placeholder text we want to use?

**Review by kkoskelin — CHANGES_REQUESTED** [review]
One small note.

**Review by sutt0n — APPROVED** [review]

File: web-client/src/views/CaseDeadlines/CaseDeadlines.jsx (hunk 2 above)
Comment (rachelschneiderman) [reply web-client/src/views/CaseDeadlines/CaseDeadlines.jsx]: Good catch, probably not!

**Review by kkoskelin — APPROVED** [review]

**Pull request merged**